src/task/index.html

```diff
@@ -106,6 +106,19 @@
             <option value="ASSIGNED">Assigned</option>
           </select>
         </div>
+        <!-- Options for AVAILABLE -->
+        <div class="inputBox" id="estimatedNoOfDaysInput">
```

Comment:
estimatedNoOfDaysInput id  name doesn't suit well for the div wrapper.

Reply:
should I name it as `estimatedNoOfDaysWrapper`?

Reply:
What we can do is, add the id to div as estimatedNoOfDays and for input as estimatedNoOfDaysInput. If you find any better name, please update.

Reply:
Okay. I'll go with your suggestion.

Reply:
`estimatedNoOfDaysInput` changed to `estimatedDays`

Reply:
Looks fine to me